keyboards/matrix/pjf/matrix.c

```diff
@@ -0,0 +1,187 @@
+/**
```

Comment:
Any reason to why custom matrix was introduced? Many f1xx boards exist in the repo, and work fine without it

Reply:
This keyboard matrix row was pulled down by external resistor. The current matrix codes can't handle such cases.